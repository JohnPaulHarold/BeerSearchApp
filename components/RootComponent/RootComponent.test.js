/**
 * @format
 */
jest.useFakeTimers();

import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import { RootComponent } from './RootComponent';

describe('RootComponent', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const render = renderer.render(<RootComponent />);
    expect(render).toBeTruthy();
    expect(render).toMatchSnapshot();
  });
});
