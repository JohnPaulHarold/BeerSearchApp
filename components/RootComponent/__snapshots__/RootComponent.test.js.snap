// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootComponent renders correctly 1`] = `
<Styled(Container)>
  <Styled(Header)>
    <Styled(Left)>
      <Styled(Button)
        transparent={true}
      >
        <Styled(Icon)
          name="menu"
        />
      </Styled(Button)>
    </Styled(Left)>
    <Styled(Body)>
      <Styled(Title)>
        Header
      </Styled(Title)>
    </Styled(Body)>
    <Styled(Right) />
  </Styled(Header)>
  <Styled(Content)>
    <Styled(Text)>
      Press Cmd+R to reload,
Cmd+D or shake for dev menu
    </Styled(Text)>
    <SearchForm />
  </Styled(Content)>
  <Styled(Footer)>
    <Styled(FooterTab)>
      <Styled(Button)
        full={true}
      >
        <Styled(Text)>
          Footer
        </Styled(Text)>
      </Styled(Button)>
    </Styled(FooterTab)>
  </Styled(Footer)>
</Styled(Container)>
`;
